airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination_async/DetectStreamToFlush.java

```diff
@@ -187,7 +187,7 @@ long estimateSizeOfRunningWorkers(final StreamDescriptor stream, final long curr
     final long workersWithBatchesSize = runningWorkerBatchesSizes.stream().filter(Optional::isPresent).mapToLong(Optional::get).sum();
     final long workersWithoutBatchesCount = runningWorkerBatchesSizes.stream().filter(Optional::isEmpty).count();
     final long workersWithoutBatchesSizeEstimate = Math.min(flusher.getOptimalBatchSizeBytes(), currentQueueSize) * workersWithoutBatchesCount;
-    return (workersWithBatchesSize + workersWithoutBatchesSizeEstimate);
+    return (workersWithBatchesSize + workersWithoutBatchesSizeEstimate) / 2;
```

Comment:
What is the context for the divide by 2?

Reply:
we were just trying to get workers to pick up batches faster. i don't think it did much. i'm going to remove it. @davinchia  thoughts?